### ISSUE TITLE
Duplicity in Product name

### ISSUE THREAD

**Issue by PetrMa** [DCourtel/Wsus_Package_Publisher#13]
Hello,
after I made an upgrade from Windows Server 2016 with WSUS server to Windows Server 2019 I have a problem with WPP. The similar issue was described here https://community.spiceworks.com/topic/2195945-wsus-package-publisher-double-entries-in-the-wpp-console

Last updates import cause that now I have 2 Products with the same Vendor Name and Product Name and I´am not able to import new other updates in bulk (every time I receive the error "Index was outside the bounds of the array") and canť delete the Product folders.
After I trying to detele the Product folders the folders are recreated with the "Delete Me!" update.

How can I fix this issue, because I can´t to make a Windows upgrade rollback and now I´a m not able to dowload new updates from Patch My PC SCUP update catalog in bulk.

Thanks, Regards,
Petr

**Comment by PetrMa** [comment]
Hello,
I fixed the issue by deleting the Vendor Name and Product Name records directly from SUSDB sql database so now everything is OK.

The records are located in **[SUSDB].[dbo].[tbLocalizedProperty]** and **[SUSDB].[dbo].[tbLocalizedPropertyForRevision**] tables

Petr

**Comment by StadtBadWurzach** [comment]
@PetrMa
You are sure that you posted the correct table names? In my case i see only translations for update categories.

**Comment by PetrMa** [comment]
Yes, the table´s names are correct. These table includes the ID and Titles for Vendor Name and Product Name.

**Comment by StadtBadWurzach** [comment]
I'm not sure, what i have to do. Can you give me an example?
Like for one of the two products where you had problems

**Comment by PetrMa** [comment]
Hi StadtBadWurzach,
in my case I needed to remove the Vendor Name **Patch My PC** and Product Name **SCUP Updates** wich cause the duplicity after I made an upgrade to Windows Server 2019. The structure is in attached screenshot.
![Capture2](https://user-images.githubusercontent.com/24729480/63581844-31a35780-c598-11e9-934d-b5590bd5b345.PNG)
So there is step-by-step how to delete these items from WPP structure:
1) Move all updates to any other structure in WPP one-by-one
2) Run this SQL query under **SUSDB** database (use the Title names which you want to delete)
`Select * from [SUSDB].[dbo].[tbLocalizedProperty] Where Title = 'SCUP Updates' or Title = 'Patch My PC'`
3) Copy the **LocalizedPropertyID** records from SQL query result (in my case the ID´s was 2306708 and 2306709 - your ID´s will be different)
![Capture3](https://user-images.githubusercontent.com/24729480/63583407-3d444d80-c59b-11e9-8f18-668b8bebeddf.PNG)
4) Run this SQL query under SUSDB database and use ID´s from previous SQL query in Delete statement (_of course - backup the SUSDB before this command_)
`Delete from [SUSDB].[dbo].[tbLocalizedPropertyForRevision] Where LocalizedPropertyID = 2306708 or LocalizedPropertyID = 2306709`
5) Run this SQL query under SUSDB database (use the Title names which you want to delete)
`Delete from [SUSDB].[dbo].[tbLocalizedProperty] Where Title = 'SCUP Updates' or Title = 'Patch My PC'`
6) Check if the requested structure was successfully deleted from WPP and create the same structure again
7) Move all updates back to original sctructure in WPP one-by-one

Hope this help you if you have the same issue.
